### ISSUE TITLE
[Bug]: OTJ ArenaDirect is detected as MH3 ArenaDirect and no picks are detected

### ISSUE THREAD

**Issue by riQQ** [unrealities/MTGA_Draft_17Lands#42]
### Describe the bug in detail

OTJ ArenaDirect is detected as MH3 ArenaDirect. Taken Cards is empty.

### Steps to reproduce this bug

Start the draft tool and start an OTJ Arena Direct event.

### Operating System

Linux

### Operating System Version

Linux Mint 20.3 Cinnamon

### Run Type

Python Script

### Python Version

Python 3.12

### Debug Log Details

```
<07/27/2024 22:46:56> - INFO - Platform: linux
<07/27/2024 22:46:56> - INFO - Scale Factor 1.0
<07/27/2024 22:46:56> - INFO - Arena Log: Searching File Path /home/<user>/Games/magic-the-gathering-arena/drive_c/users/<user>/AppData/LocalLow/Wizards Of The Coast/MTGA/Player.log
<07/27/2024 22:46:56> - INFO - Card Data Location: 
<07/27/2024 22:46:57> - INFO - Event found ArenaDirect_OTJ_Sealed_20240726
<07/27/2024 22:46:58> - INFO - Event found ArenaDirect_OTJ_Sealed_20240726
<07/27/2024 22:46:58> - INFO - Creating new draft log: /home/<user>/source/MTGA_Draft_17Lands/Logs/DraftLog_MH3_Sealed_65073553-1364-47ff-9779-a8d00ed3d8ec.log
<07/27/2024 22:46:58> - INFO - New draft detected Sealed, ['MH3']
<07/27/2024 22:46:58> - INFO - ['MH3'], Mean: 57.2, Standard Deviation: 3.6
<07/27/2024 22:46:58> - INFO - Arena Player Log Location: /home/<user>/Games/magic-the-gathering-arena/drive_c/users/<user>/AppData/LocalLow/Wizards Of The Coast/MTGA/Player.log
<07/27/2024 22:48:55> - ERROR - overlay.__update_suggest_table - 'None'
```

**Comment by FiYir** [comment]
We added a `special_events` section to the `Temp/temp_set_list.json` file to help identify Arena Open, Qualifier, and Arena Direct events. This section includes a set_code field set to `{LATEST}`, which corresponds to MH3, the latest set on the 17Lands expansion list. Some sections, such as Arena Direct, were initially completed before we had the exact event names.

I can see for your log that Arena uses the naming format `ArenaDirect_{code}_Sealed_{date}` for these events, which is also the expected format for all regular events. So, this event doesn't need to be in the `special_events` section.

You can have the app detect the OTJ Arena Direct event by following these steps.
1. Close the app while keeping Arena open.
2. Open the `Temp/temp_set_list.json` file in a text editor.
3. Delete the `ArenaDirect` section at the bottom of the `special_events` list.
4. Reopen the app.

![image](https://github.com/user-attachments/assets/227809ec-e8c7-40b6-b116-eeccf30c1a02)

**Comment by FiYir** [comment]
@riQQ Did my solution work? If so, I will make a pull request with the fix.

**Comment by riQQ** [comment]
@FiYir sorry, I didn't have to time to play another one and the logs that I had don't make the draft tool produce the line `INFO - Event found ArenaDirect_OTJ_Sealed_20240726` as they only contain the last matches of my last try but not entering the event iirc.

**Comment by FiYir** [comment]
@riQQ Thanks for the response!

You don't need to enter a new event. I can see from your debug log that the app has already created a log file for your previous event.

Please follow the steps provided, then open the app, click on `File>Open`, and select the log file from the `Logs` folder.
![image](https://github.com/user-attachments/assets/c254a6b5-bd71-4237-8ca0-e44163cead02)

**Comment by riQQ** [comment]
@FiYir thanks for the pointer. Opening this log file works properly after editing `temp_set_list.json`. Draft is detected as *OTJ Sealed* and the *Taken Cards* window opens.

**Comment by FiYir** [comment]
@riQQ Great! I will make a pull request with this fix.

**Comment by unrealities** [comment]
Addressed in v3.26
